src/types.ts

```diff
@@ -13,4 +13,42 @@
 * under the License.
 */
 
-export type HttpMethod = "GET" | "POST";
\ No newline at end of file
+// HTTP requests & responses related 
+export type HttpMethod = "GET" | "POST";
+export interface ListCount {count: number}
+export interface ResponseList<T>  {
+    nextPaginationToken?: string
+    status: Response['status'],
+    users: T[]
+}
```

Comment:
@nkshah2 In next phase, I think we should format the API's list response to be like this(replace `users` field to be `records`):
```
export interface ResponseList<T>  {
    nextPaginationToken?: string
    status: Response['status'],
    records: T[]
}
```

So, that later we can use a generic `ResponseList` for different kind of list.

Reply:
We dont need to do this

Reply:
ok